HelpSource/Classes/Pproto.schelp

```diff
@@ -48,14 +48,14 @@ code::
 (
 SynthDef(\help_playbuf, { | out=0, bufnum = 0, rate = 1, startPos = 0, amp = 0.1, sustain = 1, pan = 0, loop = 1|
 	var audio;
-	rate = rate * BufRateScale.kr(bufnum);
-	startPos = startPos * BufFrames.kr(bufnum);
+	rate = rate * BufRateScale.ir(bufnum);
+	startPos = startPos * BufFrames.ir(bufnum);
```

Comment:
Why change `.kr` to `.ir` here? 
My thinking is that an expert user will know to make this optimization, however for a less experienced one it may lead to a surprising result ("why didn't the rate update after replacing the buffer?"). I _is_ quite unlikely in this example, but as users copy-paste code from help, I'd have a slight preference to leave these as `.kr`.

Reply:
All right, I see. As this is for the granular stuff and indeed the .ir is used in the same SynthDef, let's keep the change. Thanks!